docs/core/porting/libraries.md

```diff
@@ -13,336 +13,183 @@ ms.assetid: a0fd860d-d6b6-4659-b325-8a6e6f5fa4a1
+[Developing Libraries with Cross Platform Tools](~/docs/core/tutorials/libraries.md)   
+This topic explains how to write libraries for .NET using cross-platform CLI tools.
 
-For your convenience, here is a matrix of each .NET Standard version and each specific area it runs on:
+[Additions to the *csproj* format for .NET Core](~/docs/core/tools/csproj.md)   
+This article outlines the changes that were added to the project file as part of the move to *csproj* and MSBuild.
 
-| Platform Name | Alias |  |  |  |  |  | | |
-| :---------- | :--------- |:--------- |:--------- |:--------- |:--------- |:--------- |:--------- |:--------- |
-|.NET Standard | netstandard | 1.0 | 1.1 | 1.2 | 1.3 | 1.4 | 1.5 | 1.6 |
-|.NET Core|netcoreapp|&rarr;|&rarr;|&rarr;|&rarr;|&rarr;|&rarr;|1.0|
-|.NET Framework|net|&rarr;|4.5|4.5.1|4.6|4.6.1|4.6.2|4.6.3|
-|Mono/Xamarin Platforms||&rarr;|&rarr;|&rarr;|&rarr;|&rarr;|&rarr;|*|
-|Universal Windows Platform|uap|&rarr;|&rarr;|&rarr;|&rarr;|10.0|||
-|Windows|win|&rarr;|8.0|8.1|||||
-|Windows Phone|wpa|&rarr;|&rarr;|8.1|||||
-|Windows Phone Silverlight|wp|8.0|||||||
+[Porting to .NET Core - Analyzing your Third-Party Party Dependencies](~/docs/core/porting/third-party-deps.md)   
+This topic discusses the portability of third-party dependencies and what to do when a NuGet package dependency doesn't run on .NET Core.
 
-A key thing to understand is that **a project targeting a lower version cannot reference a project targeting a higher version**. For example, a project targeting the .NET Platform Standard version 1.2 cannot reference projects that target .NET Platform Standard version 1.3 or higher. Projects **can** reference lower versions, though, so a project targeting .NET Platform Standard 1.3 can reference a project targeting .NET Platform Standard 1.2 or lower.
+## .NET Framework technologies unavailable on .NET Core
 
-It's recommended that you pick the lowest possible .NET Standard version and use that throughout your project.
+Several technologies available to .NET Framework libraries aren't available for use with .NET Core, such as AppDomains, Remoting, Code Access Security (CAS), and Security Transparency. If your libraries rely on one or more of these technologies, consider the alternative approaches outlined below. For more information on API compatibility, the CoreFX team maintains a [List of behavioral changes/compat breaks and deprecated/legacy APIs](https://github.com/dotnet/corefx/wiki/ApiCompat) at GitHub.
 
-Read more in [.NET Platform Standard Library](../../standard/net-standard.md).
+Just because an API or technology isn't currently implemented doesn't imply it's intentionally unsupported. Feel free to file an issue in the [dotnet/corefx repository issues](https://github.com/dotnet/corefx/issues) at GitHub to ask for specific APIs and technologies. [Porting requests in the issues](https://github.com/dotnet/corefx/labels/port-to-core) are usually marked with the `port-to-core` label.
```

Comment:
I looked this over. Issues for porting requests should be marked `port-to-core`, and there's no point saying that it should "usually" be the case. I can't tell the reader how to find the other issues that aren't marked `port-to-core`. The simplest way to deal with this is just to remove the word "usually." That makes the sentence a true statement (even if the request was generated internally), and the link provides the content described by the text: "Porting requests in the issues" describes the landing page content. I'm open to suggestions tho, so if you can think of a better way to express the same thought, I'm all 👂👂👂👂👂👂.